py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
+
+RUN pip install scikit-learn==0.17
```

Comment:
If you specify `--index-url https://socrata.artifactoryonline.com/socrata/api/pypi/pypi/simple` to the pip command, we can use our internal artifactory, which seems like a good habit to get into.